### PR TITLE
feat: enhance universal link experience

### PR DESCRIPTION
**Pull request by jhe-okta**
## Description:
Remove double tapping needed to launch universal link. The SIW change is based off IDX API change. Please see detail from: https://oktawiki.atlassian.net/wiki/spaces/~988051222/pages/1007784504/Design+to+Remove+Universal+Link+Extra+Button+Click+for+IOS#Proposed-change-for-BOTH-universal-link-and-custom-uri:

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Reviewers:
@haishengwu-okta @pradeepdewda-okta @aarongranick-okta 

### Issue:

- [OKTA-335169](https://oktainc.atlassian.net/browse/OKTA-335169)

### CODE REVIEW

File: test/testcafe/spec/DeviceChallengePollView_spec.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// import { _ } from 'okta';
```
Comment (pradeepdewda-okta) [inline test/testcafe/spec/DeviceChallengePollView_spec.js:1]: remove.

**Review by pradeepdewda-okta — APPROVED** [review]

**Review by haishengwu-okta — APPROVED** [review]

**Comment by haishengwu-okta** [comment]
You'd like to rebase against `5.0` ?

**Comment by jhe-okta** [comment]
> You'd like to rebase against `5.0` ?

@haishengwu-okta when is `5.0` going to rebase off `master`?

**Comment by haishengwu-okta** [comment]
@jhe-okta shall have been done. what are we missing in `5.0` atm?

**Pull request closed without merging**